client/src/app/api/register/route.ts

```diff
@@ -0,0 +1,33 @@
+'use server';
+import { NextResponse } from "next/server";
+import { API_URL } from "../../../config"
+import { responseData } from "../../../types";
+import { cookies } from "next/headers";
+
+
+export async function POST(req: Request) {
+    try {
+        const { name, email, password, contactNo } = await req.json()
+        const response = await fetch(`${API_URL}/user/create`, {
```

Comment:
use the proper name for the response coming from the fetch request.

response ==> createUserRes